hazelcast/src/main/java/com/hazelcast/cp/internal/RaftInvocationManager.java

```diff
@@ -163,7 +163,7 @@ private List<RaftEndpoint> generateRandomGroupMembers(List<CPMemberInfo> cpMembe
                         .collect(Collectors.toList());
     }
 
-    private void invokeCreateRaftGroup(String groupName, int groupSize, long groupIndex, List<RaftEndpoint> members,
+    public void invokeCreateRaftGroup(String groupName, int groupSize, long groupIndex, List<RaftEndpoint> members,
```

Comment:
maybe add a comment `// public for testing` to discourage people tempted to use it in production code?

Reply:
good point, I added there some explanation